swarm/src/handler/multi.rs

```diff
@@ -230,12 +229,12 @@ where
         }
     }
 
-    fn connection_keep_alive(&self) -> KeepAlive {
+    fn connection_keep_alive(&self) -> bool {
         self.handlers
             .values()
             .map(|h| h.connection_keep_alive())
-            .max()
-            .unwrap_or(KeepAlive::No)
+            .min()
```

Comment:
Is this correct? If any of these returns `true`, we should return `true` here.

Reply:
I misunderstand `Ord ` impl on `KeepAlive` , reverted it back 69a4a8b already